sdk/communication/communication-identity/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Features Added
 
+- Updated to `@azure/communication-common` 2.2.0.
```

Comment:
I guess we are putting the dependency changes under the title `Other Changes`

Reply:
I followed the existing pattern where the bumping of the tracing library was put under features...but I can change it I suppose...

Reply:
fixed